test/integration/integration.test.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const KafkaProducer = require('no-kafka');
+const kafkaConsumer = require('../../src/kafkaConsumer');
+const config = require('../../src/config.env')();
+
+// Heroku
+const producer = new KafkaProducer.Producer({
+  connectionString: config.connectionString,
+  ssl: {
+    cert: config.sslCert,
+    key: config.sslKey,
+  },
+});
+
+producer.init();
+
+const sendPing = async (key, value, topic) => producer.send({
+  topic,
+  partition: 0,
+  message: {
+    key,
+    value: JSON.stringify(value),
+  },
+});
+
+// TODO: Once we can enable setup Kafka on Heroku CI we would want to write
+// this integration test. As of now, there is no documentation on how to do that
+describe('test/integrtion/integration.test.js', () => {
+  it('Should receive a message from producer', async () => {
+    // const testTopic = config.topics[0];
```

Comment:
Not clear on why this doesnt work. I would assume that you can check that the consumer is getting the message when you do sendPing.

Reply:
We'd want to set up a Kafka Instance for each integration test and then tear it down after it runs the tests. Ian told me we do that for redis and postgres when we do integration testing and would wanna do the same for Kafka. Currently it can't be done, since the add-ons on app.json does not have Kafka (in the documentation). We have asked the heroku group for help on this.